.pre-commit-config.yaml

```diff
@@ -0,0 +1,21 @@
+# See https://pre-commit.com for more information
```

Comment:
can you add documentation to Readme, of how to use and install it?

Reply:
It's very cool and smart in my practice.
```
sudo pip install pre-commit
pre-commit
<update file>
git commit -sa 
<automatically check and fix>
```

We need to document that how to disable it.

Reply:
`pre-commit uninstall` to remove the hook
and 
git commit ... -n to disable the hooks

Reply:
I'll update the readme to explain those a bit